docs/en_US/install/wiiu.md

```diff
@@ -110,3 +110,41 @@ Press `B` twice, followed by the `HOME` button. The console will reboot with Pre
 Now, switch to your desired Network ID via the Account selection screen.
 
 To return to Pretendo Network, repeat the process ensuring 'Connect to the Pretendo Network' is set to **true**.
+
+# Transfering save data to your Pretendo Network account
+
+As this guide has made clear, you have separate profiles for Nintendo Network and Pretendo Network. You may want to use the save data on your Nintendo Network profile on the Pretendo Network profile, particularly for online games such as Splatoon.
```

Comment:
This line is a bit misleading, as it implies this works for online save data when that is not the case. Also I'd like to keep this guide as generic as possible, without directly mentioning specific titles unless we have to

```suggestion
Pretendo Network is not compatible with existing Nintendo Network IDs. This means you must create a new account. Because of this, you may want to move existing game save data to your new account.

<div class="tip red">
	<strong>Note:</strong>
	This only works with local save data. Any user data stored on Nintendo's servers cannot be transferred to any other accounts.
</div>
```